### PR TITLE
python3Packages.blis: 0.9.1 -> 0.7.9

### PR DESCRIPTION
**Pull request by benxiao**
###### Description of changes
https://github.com/NixOS/nixpkgs/pull/218614#discussion_r1144749894

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/blis/default.nix [hunk 1]
```diff
@@ -45,10 +45,15 @@ buildPythonPackage rec {
     "blis"
   ];
 
+  passthru = {
+    skipBulkUpdate = true;
```
Comment (mweinelt) [inline pkgs/development/python-modules/blis/default.nix:49]: Really needs to come with an explanation.

File: pkgs/development/python-modules/blis/default.nix [hunk 2]
```diff
@@ -45,10 +45,15 @@ buildPythonPackage rec {
...
+  };
```
Comment (mweinelt) [inline pkgs/development/python-modules/blis/default.nix:50]:
```suggestion
  };
```

**Review by mweinelt — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/blis/default.nix (hunk 1 above)
Comment (benxiao) [reply pkgs/development/python-modules/blis/default.nix:49]: added.

File: pkgs/development/python-modules/blis/default.nix [hunk 3]
```diff
@@ -45,10 +45,17 @@ buildPythonPackage rec {
...
+    # https://github.com/explosion/thinc/issues/771#issuecomment-1255825935
+    skipBulkUpdate = true;
+  };
+
   meta = with lib; {
     description = "BLAS-like linear algebra library";
     homepage = "https://github.com/explosion/cython-blis";
     license = licenses.bsd3;
     maintainers = with maintainers; [ ];
+    platforms = platforms.x86_64;
```
[the remark this answers is not in the document]
Comment (benxiao) [reply pkgs/development/python-modules/blis/default.nix]: I removed this line.

**Comment by mweinelt** [comment]
@ofborg build python3.pkgs.blis
@ofborg build python311.pkgs.blis

**Pull request merged**